OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -62,6 +62,19 @@ public class ArmamentInfo : UpgradableTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		[Desc("Is it possible to use the armament when force-attacking?")]
+		public readonly bool ForceAttacks = true;
+
+		[Desc("Is the armament beneficial (should it be used on allies)?")]
+		public readonly bool Beneficial = false;
```

Comment:
No, it wouldn't, as those are different things. `Beneficial` doesn't sound very good to me too. Let's try to come up with an alternative.
`IsUsedOnAllies` ?  `AutoTargetsAllies` ?